frontend/src/components/forms/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { css } from '@emotion/react';
+import { Help } from '@mui/icons-material';
+import { FormControl, FormLabel, TextField, Tooltip } from '@mui/material';
+import { DesktopDatePicker, LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import dayjs, { Dayjs } from 'dayjs';
+import React, { useState } from 'react';
+import { Controller, ControllerRenderProps, FieldValues, useFormContext } from 'react-hook-form';
+
+import { useTranslations } from '@frontend/stores/lang';
+
+interface CustomFormLabelProps {
+  label: string;
+  tooltip: string;
+  error?: boolean;
+}
+
+function CustomFormLabel({ label, tooltip, error }: CustomFormLabelProps) {
+  const [open, setOpen] = useState(true);
+
+  return (
+    <FormLabel
+      error={error}
+      css={css`
+        display: flex;
+        justify-content: space-between;
+      `}
+    >
+      {label}
+      {error ? (
```

Comment:
[nit] In react it's quite common to use "short-circuiting" in these cases, e.g. `{error && (...)}` to get rid of the ternary else `null`